core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
```

Comment:
I will run "sbt scalastyle" before adding some commits in future.

Regarding *log4j.properties.template* changes : -

I have added configuration template for metrics as log in
*log4j.properties.template,
*so other user doesn't face problem.

To Sean Owen :-

I don't understand what you meant by "Why not the class name for the logger
name here, per your JIRA?" ??

I gave generalize name for class as i didnt found any convention for it.


On Tue, Apr 26, 2016 at 2:37 PM, Hyukjin Kwon <notifications@github.com>
wrote:

> In core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala
> <https://github.com/apache/spark/pull/12697#discussion_r61051439>:
>
> > @@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
> >  import org.apache.spark.SecurityManager
> >  import org.apache.spark.metrics.MetricsSystem
> >
> > +import org.slf4j.Logger
> > +import org.slf4j.LoggerFactory
>
> (It would be great if you run sbt scalastyle before adding some more
> commits which shows up such things)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12697/files/5a437a12fb0c6cfdd7119d63bebc3f3f2c935d5f#r61051439>
>



-- 
Mihir Monani
(+91)-9429473434